Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -116,14 +140,29 @@ public final class ManagePledgeViewModel:
       .map { project, env in managePledgeSummaryViewData(with: project, envelope: env) }
       .skipNil()
 
+    let projectOrBackingFailedToLoad = Signal.merge(
+      fetchProjectEvent.map { $0.error as Error? },
```

Comment:
Also here, `fetchProjectEvent.errors()`? You shouldn't have to `.filter(isNotNil)` because `.errors()` only sends an event if there's an error.

Reply:
In this case it was to erase the different types of errors so that they could be merged.